recipients-resolver/src/main/java/com/redhat/cloud/notifications/recipients/resolver/RecipientsResolver.java

```diff
@@ -30,20 +29,24 @@ public class RecipientsResolver {
     FetchUsersFromExternalServices fetchingUsers;
 
     @Inject
-    @RestClient
-    RelationshipsApi kesselRelationshipApi;
+    KesselService kesselLookupService;
 
-    @CacheResult(cacheName = "find-recipients")
     public Set<User> findRecipients(String orgId, Set<RecipientSettings> recipientSettings, Set<String> subscribers, Set<String> unsubscribers, boolean subscribedByDefault) {
+        return findRecipients(orgId, recipientSettings, subscribers, unsubscribers, subscribedByDefault, null);
+    }
+
+    @CacheResult(cacheName = "find-recipients")
```

Comment:
I wonder if we should stop caching users here, since AuthZ is in theory going to be faster than RBAC... otherwise I guess a user that gets revoked their permission would probably receive the email because we've got the results cached, no?

Reply:
All right then!